code/modules/food_and_drinks/food/foods/meat.dm

```diff
@@ -271,6 +271,12 @@
 	return 1
 
 /obj/item/reagent_containers/food/snacks/monkeycube/proc/Expand()
+	if(LAZYLEN(SSmobs.cubemonkeys) >= config.cubemonkeycap)
+		if(fingerprintslast)
+			to_chat(get_mob_by_ckey(fingerprintslast), "<span class='warning'>Bluespace harmonics prevent the spawning of more than [config.cubemonkeycap] monkeys on the station at one time!</span>")
+		else
+			visible_message("<span class='notice'>[src] fails to expand!</span>")
+		qdel(src)
```

Comment:
you need an explicit return here.

Yes, you *can* rely on `QDELETED`, but I'd rather explicitly return.

Reply:
Now that I think about it, why does this have to delete if it fails? Seems like it would kinda suck for a cube to get deleted if you happen to bump up against the limit and aren't being a spammy jerk.

Reply:
no longer deletes on fail